org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -643,8 +643,7 @@ SsoLlmDeveloper:
     principalId: !Ref llmDeveloperGroup
     permissionSetName: 'LlmDeveloper'
     managedPolicies:
-      - 'arn:aws:iam::aws:policy/AmazonBedrockFullAccess'
-      - 'arn:aws:iam::aws:policy/AWSCloudFormationFullAccess'
+      - 'arn:aws:iam::aws:policy/PowerUserAccess'
```

Comment:
Note, this is essentially the same as [developer role](https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/700-aws-sso/_tasks.yaml#L393-L432) access, so why not just use the developer role?

Also PowerUser should be paired with a deny assume role policy to prevent unauthorized role escalation.

Reply:
The goal is to track the permissions actually used and move to more a more restricted set of permissions. Added policy to prevent assume-role (same as ssoDeveloper).